.github/workflows/push-main.yml

```diff
@@ -132,12 +132,21 @@ jobs:
           for IMAGE_TAG in $IMAGE_TAGS;
           do
             sudo skopeo copy \
-              --src-tls-verify=false \
-              --registries-conf=.github/registries.conf \
-              docker://localhost:5000/workspace-base-images:$IMAGE_TAG \
-              docker://${{ env.GAR_IMAGE_REGISTRY }}/gitpod-artifacts/docker-dev/workspace-$IMAGE_TAG:${{ env.TIMESTAMP_TAG }} &
+            --src-tls-verify=false \
+            --registries-conf=.github/registries.conf \
+            docker://localhost:5000/workspace-base-images:$IMAGE_TAG \
+            docker://${{ env.GAR_IMAGE_REGISTRY }}/gitpod-artifacts/docker-dev/workspace-$IMAGE_TAG:${{ env.TIMESTAMP_TAG }} &
```

Comment:
Why do we copy the image again (below) rather than use the `--additional-tag=latest` flag?

Reply:
[Reference](https://github.com/containers/skopeo/blob/main/docs/skopeo-copy.1.md)

Reply:
NVM, I think we'll need to do the second copy, documentation says:

`Additional tags (supports docker-archive).`